### ISSUE TITLE
scene: users with view role can move panels

### ISSUE THREAD

**Issue by baur** [grafana/scenes#203]

**Comment by torkelo** [comment]
@baur in scenes you as a developer have to control this via the isDraggable property on SceneGridLayout (it is not something the scenes framework is responsible for) 

isDraggable  should maybe not default to true however, @dprokop what do you think?

**Comment by dprokop** [comment]
> isDraggable should maybe not default to true however, @dprokop what do you think?

I agree. The default behavior in apps should be non draggable as the layout is not persisted anyhow unless developer implements something on top of scenes.

**Comment by torkelo** [comment]
Fixed by https://github.com/grafana/scenes/pull/246
